drivers.py

```diff
@@ -102,7 +102,13 @@ def __init__(self, config):
 
             # construct base url for bucket
             scheme = "https://" if config.as_bool("minio.secure") else "http://"
-            self.base_url = scheme + config.minio.endpoint + "/" + self.bucket
+
+            if config.minio.region and "amazonaws" in config.minio.endpoint.lower():
```

Comment:
seem ok to me, if there is no way to use Minio sdk BaseURL or something like that to get properly formatted URL

Reply:
There is only [get_presigned_url](https://min.io/docs/minio/linux/developers/python/API.html#get_presigned_url) but that needs specification of expiration for that url. So it does not work well for us.